platform/solr/solr-banana/src/main/resources/banana-int/Banana-Default-Dashboard.json

```diff
@@ -0,0 +1,550 @@
+{
+  "title": "Metadata Default Dashboard",
```

Comment:
Should this be `Metacard Default Dashboard`. Wondering if the applicability is beyond just the Metacard.Metadata attribute. If not, this should be fine.